tor/cmd_onion.go

```diff
@@ -3,88 +3,34 @@ package tor
 import (
 	"errors"
 	"fmt"
-	"io/ioutil"
-	"os"
-)
-
-var (
-	// ErrNoPrivateKey is an error returned by the OnionStore.PrivateKey
-	// method when a private key hasn't yet been stored.
-	ErrNoPrivateKey = errors.New("private key not found")
-)
 
-// OnionType denotes the type of the onion service.
-type OnionType int
-
-const (
-	// V2 denotes that the onion service is V2.
-	V2 OnionType = iota
-
-	// V3 denotes that the onion service is V3.
-	V3
+	"github.com/lightningnetwork/lnd/tor/onionfile"
 )
 
 // OnionStore is a store containing information about a particular onion
 // service.
 type OnionStore interface {
 	// StorePrivateKey stores the private key according to the
 	// implementation of the OnionStore interface.
-	StorePrivateKey(OnionType, []byte) error
+	StorePrivateKey(onionfile.OnionType, []byte) error
```

Comment:
If the onion type is never actually used in these methods (and we're going to remove V2 soon anyway), should we remove them in this refactor?

Reply:
Yeah if they're not used anyway, makes sense to me

Reply:
Took a stab at removing these